api/server.py

```diff
@@ -267,8 +267,11 @@ def count_all_events():
             return jsonify(
                 {
                     "count_all_events": (
-                            connector.honeypot_events.estimated_document_count() +
-                            connector.network_events.estimated_document_count()
+                            connector.honeypot_events
+                                    .estimated_document_count()
```

Comment:
readability is less than previous one although it doesn't satisfy the line chars

Reply:
I tried to fix this issue. Take a look and let me know if it improves the readability.